server/prisma/migrations/migration_lock.toml

```diff
@@ -0,0 +1,3 @@
+# Please do not edit this file manually
+# It should be added in your version-control system (i.e. Git)
+provider = "mysql"
```

Comment:
This file and `migration.sql` should not be committed! Please stash or just delete migrations if they're getting in the way of rebases or whatever